README.md

```diff
@@ -14,7 +14,7 @@ An [SBT](http://www.scala-sbt.org/) plugin for managing [Docker](http://docker.i
 
 Add the following to your `project/plugins.sbt` file:
 
-    addSbtPlugin("com.mintbeans" % "sbt-ecr" % "0.13.0")
+    addSbtPlugin("com.mintbeans" % "sbt-ecr" % "0.14.0")
```

Comment:
As a general rule, please don't change this as it's an integral part of the release process (i.e. should be updated only after the current version is published to Bintray). 

That being said, we can leave this in for now, as I'm going to release in a moment anyway.